src/applications/hca/config/chapters/veteranInformation/birthSex.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+import { genderLabels } from 'platform/static-data/labels';
+import fullSchemaHca from 'vets-json-schema/dist/10-10EZ-schema.json';
+
+import PrefillMessage from 'platform/forms/save-in-progress/PrefillMessage';
+import AdditionalInfo from '@department-of-veterans-affairs/component-library/AdditionalInfo';
+
+const { gender } = fullSchemaHca.properties;
+
+const BirthSexDescription = () => {
+  return (
+    <div className="vads-u-margin-y--2">
+      <AdditionalInfo triggerText="Why we ask for this information">
+        <p>
+          Population data shows that a person’s sex can affect things like their
+          health risks and the way their body responds to medications. Knowing
+          your sex assigned at birth, along with other factors, helps your
+          health care team use data to:
+        </p>
+
+        <ul>
+          <li>Interpret your lab results</li>
+
+          <li>Prescribe the right dose of medications</li>
+          <li>
+            Recommend health prevention screenings We also collect this
+            information to better understand our Veteran community
+          </li>
+        </ul>
+
+        <p>
+          We also collect this information to better understand our Veteran
+          community. This helps us make sure that we’re serving the needs of all
+          Veterans.
+        </p>
+      </AdditionalInfo>
+    </div>
+  );
+};
+
+export default {
+  uiSchema: {
+    'ui:description': PrefillMessage,
+    gender: {
+      'ui:title': 'What sex were you assigned at birth?',
+      'ui:description': BirthSexDescription,
+      'ui:widget': 'radio',
+      'ui:options': {
+        labels: genderLabels,
+      },
+    },
+  },
+  schema: {
+    type: 'object',
+    required: ['gender'],
+    properties: {
+      gender: {
+        ...gender,
+      },
+    },
```

Comment:
Simplify to `properties: { gender }`?

Reply:
yeahhhh was modifying it then spreading it but changed it back, thanks for the feedback. Do you know anything about the failing CI/CD @Mottie ?

Reply:
@Mottie I nuked the lock to see if it might help with CI/CD erroneous failures.

Reply:
No idea, but maybe merging in the latest master might help?

Reply:
errr, again? LOL

Reply:
I merged in master, 🤞🏻

Reply:
Hmmm... I wonder about this:
![Screen Shot 2021-11-15 at 2 20 53 PM](https://user-images.githubusercontent.com/136959/141848468-b212693e-ea55-4c95-9c72-da05c8d783f5.png)

Maybe let the support team know?